src/harness/testcases/WINNF_FT_S_REG_testcase.py

```diff
@@ -424,6 +424,185 @@ def test_WINNF_FT_S_REG_5(self):
+    device_13['cbsdSerialNumber'] = 'device_13_serial_number'
+    device_14 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    device_14['cbsdSerialNumber'] = 'device_14_serial_number'
+    device_15 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_g.json')))
+    device_15['cbsdSerialNumber'] =  'device_15_serial_number'    
+    device_16 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    device_16['cbsdSerialNumber'] =  'device_16_serial_number'
+    # Devices that contain all necessary Fcc and user data
+    devices_with_administrative_data = [device_1, device_2, device_7, device_8,\
+                                        device_9, device_15]
+    # Inject Fcc Id and User Id for the devices that contain all Fcc and user data data
+    for device in devices_with_administrative_data:
+        self._sas_admin.InjectFccId({'fccId': device['fccId']})
+        self._sas_admin.InjectUserId({'userId': device['userId']})
+    # Inject Fcc Id with not default Fcc max Eirp
+    self._sas_admin.InjectFccId({'fccId':'fccId_with_approved_eirp',\
+                                  'fccMaxEirp': 25})   
+    
+    # (Generate CPI RSA keys and) Load CPI user info
+    cpi_id = 'professional_installer_id_1'
+    cpi_name = 'a_name'
+    cpi_private_key, cpi_public_key = generateCpiRsaKeys()
+    self._sas_admin.InjectCpiUser({
+        'cpiId': cpi_id,
+        'cpiName': cpi_name,
+        'cpiPublicKey': cpi_public_key
+    })
+    # Pre-load conditionals
+    conditionals_12 = {
+        'cbsdCategory': device_12['cbsdCategory'],
+        'fccId': device_12['fccId'],
+        'cbsdSerialNumber': device_12['cbsdSerialNumber'],
+        'airInterface': device_12['airInterface'],
+        'installationParam': device_12['installationParam'],
+        'measCapability': device_12['measCapability']
+    }
+    conditionals_13 = {
+        'cbsdCategory': device_13['cbsdCategory'],
+        'fccId': device_13['fccId'],
+        'cbsdSerialNumber': device_13['cbsdSerialNumber'],
+        'airInterface': device_13['airInterface'],
+        'installationParam': device_13['installationParam'],
+        'measCapability': device_13['measCapability']
+    }
+    conditionals_14 = {
+        'cbsdCategory': device_14['cbsdCategory'],
+        'fccId': device_14['fccId'],
+        'cbsdSerialNumber': device_14['cbsdSerialNumber'],
+        'airInterface': device_14['airInterface'],
+        'installationParam': device_14['installationParam'],
+        'measCapability': device_14['measCapability']
+    }
+
+    conditionals = {
+        'registrationData': [
+            conditionals_12, conditionals_13, conditionals_14
+        ]
+    }
+    self._sas_admin.PreloadRegistrationData(conditionals)
+
+    # Remove conditionals from registration
+    del device_12['cbsdCategory']
+    del device_12['airInterface']
+    del device_12['measCapability']
```

Comment:
I think you also need to delete installationParam here and for `device_13`.

Reply:
it is deleted when we convert the request to RequestWithCpiSignature, right?

Reply:
Oops, right you are! Sorry :)